plugins/InspectionGadgets/InspectionGadgetsAnalysis/resources/messages/InspectionGadgetsBundle.properties

```diff
@@ -1658,6 +1658,9 @@ variable.not.used.inside.conditional.problem.descriptor=<code>#ref</code> checke
 boolean.expression.may.be.conditional.display.name=Boolean expression could be replaced with conditional expression
 if.may.be.conditional.problem.descriptor=<code>#ref</code> could be replaced with conditional expression #loc
 if.may.be.conditional.quickfix=Replace with conditional expression
+boolean.expression.may.be.factorized.display.name=Boolean expression could have a factorized expression
```

Comment:
I think an inspection name like "Boolean expression can be factorized" would be shorter and clearer.